### ISSUE TITLE
get_history not working for a stock

### ISSUE THREAD

**Issue by InfestusLeo** [swapniljariwala/nsepy#132]
Hi, I'm unable to fetch the data for DMART.
The code for all other stock tickers that I have tested works fine.

Snippet of the code:
tick = nse.get_history(symbol='DMART', start=start, end=end)

**Comment by avnishbm** [comment]
Also, not giving any data for following shares, where as data exists: 
JITFINFRA
PVR

I am trying for the date range 2000-01-01 till date, fetched in incremental manner 1000 days at a time i.e. multiple calls to get_history(). Can this be looked into for it to work for these shares as well? FYI, my code does work well for all other shares.

**Comment by shashankTwr** [comment]
@avnishbm Are you dynamically fetching symbol to the get_history function? Cause when I do dynamically function fails

**Comment by avnishbm** [comment]
> @avnishbm Are you dynamically fetching symbol to the get_history function? Cause when I do dynamically function fails

Yes, fetch the symbols using nsepy.symbols.get_symbol_list() function.

**Comment by shashankTwr** [comment]
> > @avnishbm Are you dynamically fetching symbol to the get_history function? Cause when I do dynamically function fails
> 
> Yes, fetch the symbols using nsepy.symbols.get_symbol_list() function.
`AttributeError                            Traceback (most recent call last)
<ipython-input-134-8da39fe339d3> in <module>
      1 import nsepy
----> 2 x=nsepy.symbols.get_symbol_list()

AttributeError: module 'nsepy' has no attribute 'symbols'`

**Comment by shashankTwr** [comment]
> > @avnishbm Are you dynamically fetching symbol to the get_history function? Cause when I do dynamically function fails
> 
> Yes, fetch the symbols using nsepy.symbols.get_symbol_list() function.

I have found a workaround by fetching the list of nifty50 stocks from their archives and then feeding them separately

**Comment by avnishbm** [comment]
> > > @avnishbm Are you dynamically fetching symbol to the get_history function? Cause when I do dynamically function fails
> > 
> > 
> > Yes, fetch the symbols using nsepy.symbols.get_symbol_list() function.
> 
> I have found a workaround by fetching the list of nifty50 stocks from their archives and then feeding them separately

This is what i had in code to import the function:
from nsepy.symbols import get_symbol_list

nsesymbols = get_symbol_list()
nsesymbols.columns = nsesymbols.columns.str.strip() // as some column names have an extra space character
